tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -395,6 +412,34 @@ public function theAdminHasCreatedUser(
 		}
 	}
 
+	/**
+	 * @When /^the user "([^"]*)" creates new user using GraphApi with settings:$/
```

Comment:
```suggestion
	 * @When /^the user "([^"]*)" creates a new user using GraphAPI with the following settings:$/
```
API is an acronym...so capital case will be more readable.